config/marketplaceApps.json

```diff
@@ -3217,5 +3217,171 @@
     ],
     "visible": true,
     "enabled": true
+  },
+  {
+    "description": "Host your PRIVATE (password protected) Simplex SMP Server w/ Clearnet IP + Onion Address",
+    "priceUSD": 6.44,
+    "multiplier": 1,
+    "category": "Hosting",
+    "version": 7,
+    "name": "PrivateSimpleXSMP",
+    "instances": 3,
+    "contacts": [],
+    "geolocation": [],
+    "staticip": true,
+    "nodes": [],
+    "isAutoEnterprise": true,
+    "expire": 22000,
+    "compose": [
+      {
+        "name": "smp",
+        "description": "SimpleX SMP Server",
+        "repotag": "simplexchat/smp-server:latest",
+        "containerPorts": [5223],
+        "domains": [""],
+        "environmentParameters": ["ADDR=127.0.0.1"],
+        "commands": [],
+        "containerData": "g:/etc/opt/simplex|0:/var/opt/simplex",
+        "tiered": false,
+        "cpu": 0.1,
+        "ram": 100,
+        "hdd": 1,
+        "ports": [5223],
+        "userSecrets": [
+          {
+            "name": "PASS",
+            "description": "Enter the password you want to use for your Private SimpleX SMP Server",
+            "placeholder": "Enter the server password you would like to use"
+          }
+        ],
+        "secrets": "",
+        "repoauth": ""
+      },
+      {
+        "name": "onion",
+        "description": "Generate permanent onion address for local SMP service",
+        "repotag": "wirewrex/hiddenonion:latest",
+        "containerPorts": [],
+        "domains": [],
+        "environmentParameters": ["SERVICE_HOST=fluxsmp_","SERVICE_PORT=5223","ONION_PORT=5223"],
+        "commands": [],
+        "containerData": "g:/var/lib/tor",
+        "tiered": false,
+        "cpu": 0.1,
+        "ram": 100,
+        "hdd": 1,
+        "ports": [],
+        "secrets": "",
+        "repoauth": ""
+      }
+    ],
+    "visible": true,
+    "enabled": true
+  },
+  {
+    "description": "Host your PUBLIC SimpleX SMP Server w/ Clearnet IP + Onion Address",
+    "priceUSD": 6.44,
+    "multiplier": 1,
+    "category": "Hosting",
+    "version": 7,
+    "name": "SimpleXSMP",
+    "instances": 3,
+    "contacts": [],
+    "geolocation": [],
+    "staticip": true,
+    "nodes": [],
+    "isAutoEnterprise": true,
```

Comment:
public verison should have this set to false and the price should be changed according.